src/web3/networkConfig.ts

```diff
@@ -101,6 +101,8 @@ interface NetworkConfig {
   web3AuthVerifier: string
   crossChainExplorerUrl: string
   getWalletTransactionsUrl: string
+  cKESTokenId: string
+  cUSDTokenId: string
```

Comment:
There's already a cusd token id here

Reply:
will update to comply with the existing format